src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx

```diff
@@ -11,16 +11,29 @@ import AppButton from '~/components/app-button/AppButton'
 import InputWithIcon from '~/components/input-with-icon/InputWithIcon'
 import CoursesFilterBar from '~/containers/find-course/courses-filter-bar/CoursesFilterBar'
 import { styles } from '~/containers/my-courses/add-course-with-input/AddCourseWithInput.styles'
+import CoursesFiltersDrawer from '~/containers/my-courses/courses-filters-drawer/CoursesFiltersDrawer'
+import useForm from '~/hooks/use-form'
+import { CourseFilters } from '~/types'
 
 const AddCourseWithInput = () => {
   const [inputValue, setInputValue] = useState<string>('')
+  const [isFiltersOpen, setIsFiltersOpen] = useState(false)
```

Comment:
You can use our useDrawer hook and destructure onOpen, onClose and isOpen from it

Reply:
thanks, i used it